src/utils/schemaTree.js

```diff
@@ -22,6 +22,9 @@ import { COMBINATION_TYPES, REF_TYPE, CUSTOM_KEYWORDS } from './constants';
  *                         (a node object as illustrated above)
  *   isExpanded: ..     // whether the node is expanded or not
  * }
+ *
+ * @param {object} schema
+ * @param {array} path path to current node/subtree

```

Comment:
Could you put an example of a path here, please? :-)

Reply:
Applied changes in #101 ! (Also further examples of path are also included in the comment above :)